test/test_optim.py

```diff
@@ -467,13 +467,17 @@ def test_lbfgs(self):
             wrap_old_fn(old_optim.lbfgs)
         )
         self._test_rosenbrock(
-            lambda params: optim.LBFGS(params, lr=5e-2, max_iter=5),
-            wrap_old_fn(old_optim.lbfgs, learningRate=5e-2, maxIter=5)
+            lambda params: optim.LBFGS(params, lr=1, max_iter=5),
+            wrap_old_fn(old_optim.lbfgs, learningRate=1, maxIter=5)
```

Comment:
conditions change, so the test is not the same as before. Anyway, with lr=1, the test passed.

Reply:
Could you please elaborate on "why are you increasing the learning rate"?

Reply:
This test compares to the `lua` version. I make some small changes based on the updated version of `minFunc`, so the results of `pytorch` and `lua` versions are a little bit different.